content/applications/inventory_and_mrp/barcode/setup/device_troubleshooting.rst

```diff
@@ -0,0 +1,212 @@
+
+Begin on the Zebra scanner's home screen, and select the :guilabel:`DataWedge` app (the app is
+represented by a :guilabel:`(light blue barcode)` icon).
+
+On the :guilabel:`DataWedge Profiles` page, select the profile option to access the Zebra scanner's
+settings.
+
+Once the profile is selected, scroll down to the :guilabel:`Keyboard Output` option, and ensure the
+:guilabel:`Enable/disable keystroke output` option is :guilabel:`Enabled`.
+
+.. image:: device_troubleshooting/device-troubleshooting-zebra-settings.png
+   :align: center
+   :alt: Show keystroke option in the Zebra scanner's DataWedge app.
+
+Once that option is enabled, go back to the :guilabel:`Profile` options page, and go to the
+:guilabel:`Keystroke output` section. Then, open the :guilabel:`Key event options` submenu. Under
+:guilabel:`Characters`, ensure the :guilabel:`Send Characters as Events` option is checked.
+
+.. important::
+   The :guilabel:`Send Characters as Events` option **must** be checked on the Zebra scanner, or
+   Odoo **cannot** recognize the barcodes that are scanned.
+
+Once the above steps have been taken, perform a test scan to ensure the Zebra scanner is working as
+intended.
+
+MUNBYN Android devices
+----------------------
+
+When using MUNBYN Android scanners, ensure the following configurations are set to prevent errors.
+
+From the device's home screen, click :menuselection:`AppSettings`. On the resulting page, locate the
+:guilabel:`Process mode` section, and select :guilabel:`Keyboard input`.
+
+.. image:: device_troubleshooting/device-troubleshooting-munbyn-process-mode.png
+   :align: center
+   :alt: Process mode section on MUNBYN scanner's AppSettings page.
+
+.. tip::
+   The selected *Process mode* controls how data is processed after barcode data has been read out.
+
+   *Keyboard input* enters read-out data at the position of the cursor, the same as input data on an
+   analogue keyboard would.
+
+Once the above steps have been taken, perform a test scan to ensure the MUNBYN Android scanner is
+working as intended.
+
+.. admonition:: Why is there no data output in the app after a successful scan?
+
+   When scanning a barcode, the scanner might beep, indicating a successful scan, but there is no
+   data output in the app.
+
+   To fix this issue, adjust the output method to *keyboard analogue* in the *Scanner* app on the
+   device.
+
+   From the device's home screen, click :menuselection:`Scanner App --> Settings`. From the
+   :guilabel:`Settings` page, click :guilabel:`Output Mode`. The resulting pop-up window presents
+   the different output options available to users. Select :guilabel:`Keyboard Mode`, then click
+   :guilabel:`OK`.
+
+   .. image:: device_troubleshooting/device-troubleshooting-output-mode-popup.png
+      :align: center
+      :alt: Output mode pop-up window on MUNBYN scanner.
+
+   Go back to the app that needs to be scanned, and click on the :guilabel:`input dialog box` first
```

Comment:
Just want to double check this– is there text/button on-screen at this stage that says "input dialog box"? If so, this :guilabel: can stay. 

Otherwise, if the instruction is to just "click on the input dialog box", no :guilabel: is needed :)
```suggestion
   Go back to the app that needs to be scanned, and click on the :guilabel:`input dialog box` first
```

Reply:
Suggestion implemented ✅